### ISSUE TITLE
Model deployment - expiration times

### ISSUE THREAD

**Issue by cabhishekin** [SAP-samples/data-attribute-recommendation-postman-tutorial-sample#11]
Hi juliana, 
I am writing with reference to your post on "Data Attribute Recommendation", I understand the metering aspect of Machine learning ([https://help.sap.com/docs/Data_Attribute_Recommendation/105bcfd88921418e8c29b24a7a402ec3/1e093326a2764c298759fcb92c5b0500.html], but I am running a free trail of BTP and exploring applications of ML 

What would you recommend while running a demo, should i deploy the model 1 hr before the demo and then use "Classify Records" to predict the labels ? Once the demo is over undeploy models and delete dataset and schema? 

Have some more questions more specific on data science and would raise them in another issue. Hope thats fine with you. 
regards
abhishek

**Comment by Juliana-Morais** [comment]
Hi Abhishek,

When using a trial account (or a productive account and the free tier service plan), you should observe the following limitations: https://help.sap.com/docs/Data_Attribute_Recommendation/105bcfd88921418e8c29b24a7a402ec3/c03b561eea1744c9b9892b416037b99a.html.

All Data Attribute Recommendation tutorials (https://developers.sap.com/tutorial-navigator.html?tag=software-product%3Atechnology-platform%2Fsap-ai-business-services%2Fdata-attribute-recommendation) were created targeting trial account (and free tier service plan) users. If you follow the steps in the suggested order, including the "delete" steps you should be fine and shouldn't be blocked by any limitation.

The Data Attribute Recommendation blog posts might help you with your machine learning questions: https://blogs.sap.com/tags/73554900100800002858/.

Best regards,
Juliana
